src/common/utils/FocusManager.ts

```diff
@@ -110,8 +110,14 @@ export abstract class FocusManager {
                 withinRestrictionOwner = true;
             }
 
-            if (parent._isFocusLimited) {
-                storedComponent.limitedCount++;
+            const isFocusLimited = parent._isFocusLimited;
```

Comment:
_isFocusLimited is no longer a boolean, so this should be:
 = (parent._isFocusLimited !== Types.LimitFocusType.Unlimited)